client/v3/concurrency/session.go

```diff
@@ -115,9 +126,10 @@ func (s *Session) Close() error {
 }
 
 type sessionOptions struct {
-	ttl     int
-	leaseID v3.LeaseID
-	ctx     context.Context
+	ttl                    int
+	leaseID                v3.LeaseID
+	ctx                    context.Context
+	sessionCreationTimeout int
```

Comment:
Suggest to remove session prefix because the struct is named by sessionOptions already.